pkg/rbd/rbd_journal.go

```diff
@@ -163,19 +163,19 @@ volume names as requested by the CSI drivers. Hence, these need to be invoked on
 respective CSI snapshot or volume name based locks are held, as otherwise racy access to these
 omaps may end up leaving the omaps in an inconsistent state.
 */
-func checkVolExists(ctx context.Context, rbdVol *rbdVolume, cr *util.Credentials) (bool, error) {
-	err := validateRbdVol(rbdVol)
+func (rv *rbdVolume) Exists(ctx context.Context) (bool, error) {
```

Comment:
it does not look like any of the other functions in this file are methods. Also, does it have to be a public function? It wasn't one before.

Reply:
It is slowly moving to be more object oriented, more functions are rewritten into methods.
I can rename it to `exists()` and make it private, but there are no common guidelines in the project to make a function public/private. Everything related to RDB is currently in the rbd package, including the provisioner/attacher. At one point these may get split a little more to divide the CSI application and internal details.

Reply:
OK, I just am concerned that we get in the habit of uppercasing things rather than starting private (lowercase) and then only making what's truly needed public. I don't have strong feelings about it, I just noticed the change.

Reply:
I prefer to keep it like this, and separate the attacherl, provisioner and `rbdVolume` into their own packages at one point.

Reply:
OK, that sounds fine. This was more of a have-you-considered than this-is-an-issue anyway. :-)